hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/impl/CryptoPreTransactionHandlerImpl.java

```diff
@@ -121,7 +121,8 @@ public TransactionMetadata preHandleUpdateAccount(TransactionBody txn) {
     @Override
     /** {@inheritDoc} */
     public TransactionMetadata preHandleCryptoTransfer(TransactionBody txn) {
-        throw new NotImplementedException();
+        final var payer = txn.getTransactionID().getAccountID();
+        return new SigTransactionMetadata(accountStore, txn, payer);
```

Comment:
We should be doing all the checks currently present at SigRequirements checks [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/services/sigs/order/SigRequirements.java#L674)